packages/nc-gui-v2/components/dashboard/settings/UserManagement.vue

```diff
@@ -0,0 +1,287 @@
+<script setup lang="ts">
+import { useDebounce } from '@vueuse/core'
+import KebabIcon from '~icons/ic/baseline-more-vert'
+import UsersModal from './userManagement/UsersModal.vue'
+import { projectRoleTagColors } from '~/utils/userUtils'
+import MidAccountIcon from '~icons/mdi/account-outline'
+import ReloadIcon from "~icons/mdi/reload"
+import MdiEditIcon from '~icons/ic/round-edit'
+import SearchIcon from '~icons/ic/round-search'
+import { copyTextToClipboard } from '~/utils/miscUtils'
+import MdiDeleteOutlineIcon from '~icons/mdi/delete-outline'
+import EmailIcon from '~icons/eva/email-outline'
+import MdiPlusIcon from '~icons/mdi/plus'
+import MdiContentCopyIcon from '~icons/mdi/content-copy'
+import MdiEmailSendIcon from '~icons/mdi/email-arrow-right-outline'
+import RolesIcon from '~icons/mdi/drama-masks'
+import { User } from '~~/lib/types'
+import { useToast } from 'vue-toastification'
+const toast = useToast()
+
+const { $api, $e } = useNuxtApp()
+const { project } = useProject()
+
+let users = $ref<null | Array<User>>(null)
+let selectedUser = $ref<null | User>(null)
+let showUserModal = $ref(false)
+let showUserDeleteModal = $ref(false)
+
+let totalRows = $ref(0)
+const currentPage = $ref(1)
+const currentLimit = $ref(10)
+const searchText = ref<string>('')
+const debouncedSearchText = useDebounce(searchText, 300)
+
+const loadUsers = async (page = currentPage, limit = currentLimit) => {
+  try {
+    console.log('loadUsers', page, limit)
+    if (!project.value?.id) return
+
+    const response = await $api.auth.projectUserList(project.value?.id, <any> {
+      query: {
+        limit,
+        offset: searchText.value.length === 0 ? (page - 1) * limit : 0,
+        query: searchText.value,
+      },
+    })
+    console.log('userData', response)
+    if (!response.users) return
+
+    totalRows = response.users.pageInfo.totalRows ?? 0
+    users = response.users.list as Array<User>
+  } catch (e) {
+    console.log(e)
+  }
+}
+
+const inviteUser = async (user) => {
+  try {
+    if (!project.value?.id) return
+
+    await $api.auth.projectUserAdd(project.id, user);
+    toast.success('Successfully added user to project');
+    await loadUsers();
+  } catch (e) {
+    toast.error(e.response.data.msg);
```

Comment:
Similarly, cast `e` to `any` and print the error using `await extractSdkResponseErrorMsg(e)`